psi4/driver/procrouting/proc.py

```diff
@@ -54,6 +54,15 @@
 from . import mcscf
 from . import response
 
+_have_cppe = False
```

Comment:
Doesn't the same portion of code in `solvent/__init__.py` suffice?

Reply:
Yes. Done.